ProcessMaker/processmaker
# Initial Implementation of VarFinder Panel

This PR ensures the `VarFinder.js` file is dynamically injected into the header of all pages when the `package-variable-finder ` package is installed. This enables users to trigger and display the VarFinder panel using established keyboard shortcuts.

## Changes
- Added a conditional check to verify if the package-variable-finder is installed.
- If installed, the VarFinder.js file is injected into the page header.

## How to Test
1. Ensure the `package-variable-finder` package on branch `subtask/FOUR-20763` is installed in your environment.
2. Follow the testing instructions outlined in the [Variable Finder PR](https://github.com/ProcessMaker/package-variable-finder/pull/2).

## Related Tickets & Packages
- [Variable Finder Package PR](https://github.com/ProcessMaker/package-variable-finder/pull/2)
- [FOUR-20692](https://processmaker.atlassian.net/browse/FOUR-20692) and its related subtasks

ci:package-variable-finder:subtask/FOUR-20763

## Code Review Checklist


[FOUR-20692]: https://processmaker.atlassian.net/browse/FOUR-20692?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
